crates/bevy_ecs/src/schedule/executor/mod.rs

```diff
@@ -21,7 +21,7 @@ pub(super) trait SystemExecutor: Send + Sync {
     fn run(
         &mut self,
         schedule: &mut SystemSchedule,
-        skip_systems: Option<FixedBitSet>,
+        skip_systems: Option<&FixedBitSet>,
```

Comment:
Nit, I'd like if the skip_systems argument came after world since it's conditionally used. No big deal either way though.

Reply:
Yea, that makes sense.  I swapped them.